### ISSUE TITLE
CKS base register wrong in svd. Overlaps with UART0

### ISSUE THREAD

**Issue by btashton** [bouffalolab/bl_iot_sdk#28]
I could not find much information on what the CKS is but the base address defined in the svd overlaps with UART0 at 0x4000a000.  What is the correct base address for this?

**Comment by shchen-Lab** [comment]
Hi btashton,

UART0 base address is 0x4000a000. Please refer BL602/604 Datasheet and BL602/604 Reference Manual.

https://dev.bouffalolab.com/media/doc/602/open/datasheet/en/html/content/Functional.html
https://dev.bouffalolab.com/media/doc/602/open/reference_manual/en/html/index.html

**Comment by btashton** [comment]
Yes I know.  The base address for CKS is also listed as `0x4000a000` which cannot be correct.

**Comment by shchen-Lab** [comment]
Yes, Thanks for your help. CKS base address wrong in svd, we will fix this issue. You can get clock register information at glb_reg.h.
![1609130646(1)](https://user-images.githubusercontent.com/75290921/103190293-38b84380-490b-11eb-80f3-4b54c2a47b3e.jpg)

**Comment by mkroman** [comment]
The base address for `CKS` in an updated SVD I got from Bouffalo Lab is `0x4000A700`.

**Comment by mkroman** [comment]
I've improved the SVD [here](https://github.com/sipeed/bl602-pac/blob/main/soc602_reg.svd) and I've written some peripheral details [here](https://github.com/pine64/bl602-docs/blob/2251903f8a8eadbbf59299e4fa57acc0bc112b07/hardware_notes/registers/cks.md)
